tests/transport/spdp/spdp_transport.cpp

```diff
@@ -251,12 +254,29 @@ struct ReactorTask : ACE_Task_Base {
   }
 };
 
+void get_local_ip(OpenDDS::DCPS::OctetArray16 address)
+{
+  const String hostname = get_fully_qualified_hostname();
+  const ACE_INET_Addr addr = choose_single_coherent_address(hostname + ":54321", false);
+  std::memset(address, 0, 12);
+  *(ACE_UINT32*)(&address[12]) = ntohl(addr.get_ip_address());
```

Comment:
The address returned can be IPv6, so you can't rely on get_ip_address(). See other comment.

Reply:
removed call to get_ip_address